_includes/head.html

```diff
@@ -10,21 +10,21 @@
 
 <meta name="viewport" content="width=device-width, initial-scale=1.0, maximum-scale=1.0, user-scalable=no">
 
-<link rel="stylesheet" href="/assets/css/style.css">
+<link rel="stylesheet" href="{{ site.baseurl }}/assets/css/style.css">
 <!-- jQuery -->
-<script src="/assets/js/jquery-1.11.1.min.js"></script>
+<script src="{{ site.baseurl }}/assets/js/jquery-1.11.1.min.js"></script>
 
 <!-- If the query includes 'print-pdf', include the PDF print sheet -->
 <script>
 	if( window.location.search.match( /print-pdf/gi ) ) {
 		var link = document.createElement( 'link' );
 		link.rel = 'stylesheet';
 		link.type = 'text/css';
-		link.href = '/reveal.js/css/print/pdf.css';
+		link.href = '/{{ site.baseurl }}reveal.js/css/print/pdf.css';
```

Comment:
This causes an error. It should be `link.href = '{{ site.baseurl }}/reveal.js/css/print/pdf.css';`